src/analyze/analyze-verify.c

```diff
@@ -115,7 +115,7 @@ static int verify_socket(Unit *u) {
         return 0;
 }
 
-int verify_executable(Unit *u, const ExecCommand *exec) {
+int verify_executable(Unit *u, const ExecCommand *exec, const char *root) {
```

Comment:
the change to pass `root` to `find_executable_full` here is missing - forgot to git add before pushing after the refactor maybe?